airflow/utils/dot_renderer.py

```diff
@@ -19,9 +19,14 @@
 """Renderer DAG (tasks and dependencies) to the graphviz object."""
 from __future__ import annotations
 
+import warnings
 from typing import TYPE_CHECKING, Any
 
-import graphviz
+try:
+    import graphviz
```

Comment:
Same like comment above with FAB - the changes in this file are a candidate to separate-out to another PR

Reply:
I've explained it more in the commend description - this change makes it **actually** possible to install things easily. but yes - this is a candidate to extract and merge separately before we merge that one.

I will leave the comment open to remind me about it :)

Reply:
Extracted making graphviz as optional to a separate PR: https://github.com/apache/airflow/pull/36647